README.md

```diff
@@ -28,6 +28,15 @@ The library is broken up into a number of atomically includable Hash extension m
 
 Any of the extensions listed below can be mixed into a class by `include`-ing `Hashie::Extensions::ExtensionName`.
 
+## Logging
+
+Hashie has a built-in logger that you can override. By default, it logs to `$stdout` but can be replaced by any `Logger` class. The logger is accessible on the Hashie module, as shown below:
```

Comment:
I flip flop on this as well, though I think I prefer STDOUT now that you mention it. I think I'll update the PR to use that.